1.8/administration/virtual-networks/ip-per-container.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: page
+nav_title: IP-per-Container
+post_title: Configuring IP-per-Container in Virtual Networks
+menu_order: 10
+---
+
+In Enterprise DC/OS, the virtual network feature is enabled by default. The default configuration provides two overlay networks `dcos-1` and `dcos-2`, whose YAML configuration is as follows:
+
+```yaml
+  dcos_overlay_network:
+    vtep_subnet: 192.15.0.0/20
+    vtep_mac_oui: 70:B3:D5:00:00:00
+    overlays: 
+      - name: dcos-1
+        subnet: 192.168.0.0/17
+        prefix: 24
+      - name: dcos-2
+        subnet: 192.168.128.0/17
+        prefix: 24
+```
+
+**Note:** To use virtual networks in DC/OS you MUST make sure to use a recent Linux kernel (3.9 or above) as well as Docker version 1.11 on the agent nodes.
+
+Each overlay network is identified by a canonical `name`. Containers launched on an overlay network will get an IP address from the `subnet` allocated to the overlay network. To remove the dependency on a global IPAM (IP address management system), the overlay `subnet` is further split into smaller subnets, with each subnet being allocated to an agent. The agents can then use a host-local IPAM to allocate IP addresses from their respective subnets to containers launched on the agent and attached to the given overlay. The `prefix` determines the size of the subnet (carved the overlay `subnet`) allocated to each
```

Comment:
These sections were a written independently. So I am guess a bit of duplication in the heat of meeting 1.8 EA deadlines :).